### DISCUSSION TITLE
How many of each 🪙 did I find?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1092] [Galaxy Trivia] [answered]
Every year, just before my birthday 🎂, I open my money box 💰 to see how much money I have saved.  
Imagine my surprise when I found exactly £49.58 made up of equal numbers of 2p, 5p, 10p and 20p.

**Comment by peterbaccaro** [comment]
**134** of each coin.

There are equal numbers of 2p, 5p, 10p, and 20p.

Therefore, one of each coin would be 2p + 5p + 10p + 20p = 37p.

The money box contained £49.58 so in pence this would be 4958p.

Therefore the number of each coin can then be calculated by dividing the number of pence in the money box by the value of the sum of one of each coin.

4958 / 37 = **134**

**Comment by thuongtruong1009** [comment, marked as answer]
Let's denote the number of each coin as follows:

Number of 2p coins = x
Number of 5p coins = x
Number of 10p coins = x
Number of 20p coins = x

Given that the total amount is £49.58, we can write the equation:

2x + 5x + 10x + 20x = 4958 (converting pounds to pence)

Simplifying the equation:

37x = 4958
x = 4958 / 37
x = 134

So, you found 134 of each coin:

Number of 2p coins = 134
Number of 5p coins = 134
Number of 10p coins = 134
Number of 20p coins = 134
